ccapi/config/configsdk-org1.yaml

```diff
@@ -49,7 +49,7 @@ client:
 #
 channels:
   # name of the channel
-  mainchannel:
+  mychannel:
```

Comment:
This change is not compatible with the rest of the repo. It must be changed back

Reply:
Done